flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/rel/LogicalWindowAggregate.scala

```diff
@@ -18,8 +18,6 @@
 
 package org.apache.flink.table.plan.logical.rel
 
-import java.util
```

Comment:
Thanks, both solutions look better than the half package import. I will apply one of those.